mne/_fiff/pick.py

```diff
@@ -687,6 +688,15 @@ def pick_info(info, sel=(), copy=True, verbose=None):
     if info.get("custom_ref_applied", False) and not _electrode_types(info):
         with info._unlock():
             info["custom_ref_applied"] = FIFF.FIFFV_MNE_CUSTOM_REF_OFF
+    # remove unused projectors
+    if info.get("projs", []):
```

Comment:
I stumbled over this line, because seeing the `[]` in `foo.get("bar", [])` made me expect that the result was being assigned or iterated over. Here we're only (effectively) passing it to `bool()` for the duration of this one line; so I think it's clearer to default to `False` rather than a (falsey) empty list.

```suggestion
    if info.get("projs", False):
```